react/features/filmstrip/functions.web.js

```diff
@@ -119,9 +118,7 @@ export function isVideoPlayable(stateful: Object | Function, id: String) {
     const tracks = state['features/base/tracks'];
     const participant = id ? getParticipantById(state, id) : getLocalParticipant(state);
     const isLocal = participant?.local ?? true;
-
-    const videoTrack
-        = isLocal ? getLocalVideoTrack(tracks) : getTrackByMediaTypeAndParticipant(tracks, MEDIA_TYPE.VIDEO, id);
+    const videoTrack = getVideoTrackByParticipant(state, participant);
```

Comment:
Here I'm not sure how we could possibly end up with `participant === undefined` but right now this case is not handled. Why it is safe to do this now and it was not safe before?

Reply:
How is it safe to assume that it's the local track when participant is undefined ? Also now since we can have 2 local tracks associated with a local participant but `getLocalVideoTrack(tracks)` return only the first one. So the previous logic didn't make sense to me.

Reply:
I didn't see your above comment how a local participant can have a dummy id. Ok, it makes sense to revert the change to audioTrack lookup but `getLocalVideoTrack(tracks)` wouldn't work for the reason I mentioned above.